clever.go

```diff
@@ -225,6 +225,19 @@ type Section struct {
 	Term
 }
 
+type EventResp struct {
+	Links []Link
+	Event Event `json:"data"`
+	Uri   string
+}
+
+type Event struct {
+	Type    string
+	Created string
+	Id      string
+	Data    map[string]interface{}
```

Comment:
I thought about that and chickened out because I couldn't think of a decent name for `Object`, but if you think calling it `Object` works then :+1: